### DISCUSSION TITLE
Ideal website

### DISCUSSION THREAD

**Discussion by McJul** [NAIST-SE/AttractingContributors#371] [Ideas]
Tell me please you ideas about an ideal website for you as a customer. What would you start with? How to build a perfect website?

**Comment by ellielo** [comment]
I would find a professional developer first.

**Comment by janply** [comment]
Yeah, I agree! But sometimes, when you have your vision for your ideal website, you can manage to build it yourself. Consider about [magento vs shopify](https://forbytes.com/blog/magento-vs-shopify/). They both are really great platforms. As for the ideal website, you should be oriented on your clients first. It should be simple to use and navigate, optimized for mobile, and optimized for search and social web.
